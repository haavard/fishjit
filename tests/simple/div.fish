v
\a5,nao
\5a,:nao
\2$,:nao
\4,nao;
