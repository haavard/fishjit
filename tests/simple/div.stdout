2
0.500000
4.000000
1.000000
